### DISCUSSION TITLE
shift selection

### DISCUSSION THREAD

**Discussion by pippo73** [immich-app/immich#14278] [General]
Hi all, 
may I ask you why in the last release 1.121.0 it was implemented the shift selection only in the search page and not in all the pages?
For example I think it could be confertable to have it in the main page in order to select many photo in order to moove them in albums.
Anyway thank you for your great software :-)

**Comment by bo0tzz** [comment]
Shift select on the main page was already supported and still works.

**Reply by pippo73** [reply]
> Shift select on the main page was already supported and still works.

Ops, it seems I had a glitch on my pc, and it was not working. After a reboot everything is fine now :-) thank you for the support :-)
